### PR TITLE
Atmoslathe

### PR DESCRIPTION
**Pull request by j-mie**
:cl: Jambread/Carbonhell
add: Adds a new autolathe called atmoslathe, found in Atmospherics, to make tanks and atmos-related items.
/:cl:
Doesn't work atm, not sure why - anyone got ideas?

It's not spawning on the map and it's also not showing designs. WHO THE FUCK DIDN'T USE THE MERGER. COME ON

### CODE REVIEW

**Comment by KayeArray** [comment]
I don't see the issue with this code, it's damn near identical to the rest of the machine code

File: code/__DEFINES/~hippie_defines/machines.dm [hunk 1]
```diff
@@ -0,0 +1,3 @@
+//see: https://github.com/HippieStation/HippieStation/blob/atmoslathe/code/__DEFINES/machines.dm
```
Comment (Chnkr) [inline code/__DEFINES/~hippie_defines/machines.dm]: this is bad, and you know why

**Comment by GuyonBroadway** [comment]
Sorry to make you have to fix these conflicts but on the PLUS side the map is now properly mapmerged.

**Comment by KayeArray** [comment]
I doubt this is going to be merged anytime soon considering the situation. Someone open a new PR if you're interested.

**Pull request closed without merging**